src/agent/testrunpublisher.ts

```diff
@@ -123,19 +151,19 @@ export class TestRunPublisher {
     // - testrunID: number - runId against which results are to be published 
     // - testRunResults: TestRunResult[] - testresults to be published  
     //-----------------------------------------------------
-    public addResults(testRunId: number, testResults: testifm.TestResultCreateModel[]) : Q.Promise<testifm.TestCaseResult[]> {
+    public addResults(testRunId: number, testResults: testifm.TestResultCreateModel[]): Q.Promise<testifm.TestCaseResult[]> {
         var defer = Q.defer();
         var _this = this;
 
         var i = 0;
-        var batchSize = 100; 
+        var batchSize = 100;
```

Comment:
why 100? this will result in more server calls. I think the Windows  PublishResults task has a batchsize of 500/1000.